.github/workflows/main.yml

```
@@ -20,6 +20,15 @@ jobs:
           python -m pip install tox
       - name: Run tests
         run: tox
+      - name: Install coveragepy tool
+        run: pip install coverage
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          flags: >-
+            GHA,
+            ${{ runner.os }},
+            ${{ matrix.python }}
```

Looks like I was wrong about the root cause of the missing report. In the earlier config (pre https://github.com/pypa/setuptools/pull/2486), the CIs created a junit-compatible XML file and now they don't. Here's how to recover this:
```suggestion
        run: tox -- --junit-xml=.test-results/pytest/cov.xml
      - name: Upload coverage to Codecov
        uses: codecov/codecov-action@v1
        with:
          file: .test-results/pytest/cov.xml  # Explicitly point at the correct coverage report
          flags: >-  # Mark which lines are covered by which envs
            GHA,
            ${{ runner.os }},
            ${{ matrix.python }}
          functionalities: gcov  # disable C coverage autodiscovery
```